engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -142,6 +146,17 @@ public String get() {
                 }
             });
         }
+
+        UILabel debugLine5 = find("debugLine5", UILabel.class);
+        if (debugLine5 != null) {
+            debugLine5.bindText(new ReadOnlyBinding<String>() {
+                @Override
+                public String get() {
+                    return String.format("Executed final node task: %s", ((NodeTask) renderTaskSystem.getCursorTask()).getNode().getClass().getSimpleName());
```

Comment:
I'd suggest the string can just be

"Last node processed: %s"

because from a debugging perspective it's the nodes that will matter, not the tasks.